src/components/Collapsible/CollapsibleContent/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { useContext } from 'react'
+import { StyledCollapsibleContent } from './style'
+import { CollapsibleContext } from '../../../contexts/CollapsibleContext'
+import { CollapsibleContentProps } from './types'
+
+const CollapsibleContent = ({
+  children,
+  ...props
+}: CollapsibleContentProps) => {
+  const { isCollapsed } = useContext(CollapsibleContext)
+
+  return (
+    <StyledCollapsibleContent $open={isCollapsed} {...props}>
```

Comment:
rename prop `$open` to `open`. Do not need to add dolar sign

Reply:
In this case it is necessary because it is a styled components prop.
If I try to remove the "$" it triggers a type error.